### PR TITLE
Update link to correct content

### PR DESCRIPTION
**Pull request by rajbos**
### Why:

The link that I changed needs to point to the query parameters page [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request), since the information needed is not in the currently linked page (had me looking for a while to find it!). 

I could not figure out how the `article` part of the url works, so I thought it would be good to start the conversation 😉. It could also be valid to update the target page, but that currently only lists the query parameters for issues, and this is a separate page. 

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Update url to the correct page.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by rajbos** [comment]
The new link does work though: just tested it ❤️ .

**Comment by cmwilson21** [comment]
👋 Hey @rajbos! Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ You can create default pull request templates for your organization{% ifversion
...
+    - To create multiple pull request templates and use the `template` query parameter to specify a template to fill the pull request body, type *.github/PULL_REQUEST_TEMPLATE/*, then the name of your pull request template. For example, `.github/PULL_REQUEST_TEMPLATE/pull_request_template.md`. You can also store multiple pull request templates in a `PULL_REQUEST_TEMPLATE` subdirectory within the root or `docs/` directories. For more information, see "[About automation for pull requests with query parameters](/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request)."
```
Comment (mchammer01) [inline content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository.md:38]: Your link is perfect, but we should use the article title in the linked text 😃 
```suggestion
    - To create multiple pull request templates and use the `template` query parameter to specify a template to fill the pull request body, type *.github/PULL_REQUEST_TEMPLATE/*, then the name of your pull request template. For example, `.github/PULL_REQUEST_TEMPLATE/pull_request_template.md`. You can also store multiple pull request templates in a `PULL_REQUEST_TEMPLATE` subdirectory within the root or `docs/` directories. For more information, see "[Using query parameters to create a pull request](/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request)."
```

**Review by mchammer01 — COMMENTED** [review]
@rajbos 👋🏻 - thanks for contributing to improve our docs ✨ 
I've made a small suggestion that I'm going to apply, and then I'll merge this PR.

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**